presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -75,7 +75,6 @@
     public static final String AGGREGATION_OPERATOR_UNSPILL_MEMORY_LIMIT = "aggregation_operator_unspill_memory_limit";
     public static final String OPTIMIZE_DISTINCT_AGGREGATIONS = "optimize_mixed_distinct_aggregations";
     public static final String LEGACY_ROUND_N_BIGINT = "legacy_round_n_bigint";
-    public static final String LEGACY_JOIN_USING = "legacy_join_using";
```

Comment:
I think this should go to separate PR. It is not a cleanup, but change in behaviour.

Reply:
Yeah, I’ll pull it out to a separate PR soon (we’re removing that feature for the next release anyway). I just had it here because other pending changes I’m working on depend on it.